sdk/src/beta9/abstractions/output.py

```diff
@@ -21,13 +24,62 @@
 
 class Output(BaseAbstraction):
     """
-    A file that a task has created.
-
-    Use this to save a file you may want to save and share later.
+    A file or directory to persist after a task has ended.
     """
 
+    _tmp_dir = Path("/tmp/outputs")
```

Comment:
`_tmp_dir` is a class variable used as a base path for temp storage of all `Output`s. 

In this PR, we build temp paths with the `id` of the instance so that when the Output instance is garbage collected, we only delete that instance's temp data. This includes zipped directories and PIL.Image files.